forge/test/operators/pytorch/eltwise_binary/test_pytorch_binary.py

```diff
@@ -0,0 +1,504 @@
+        return output
+
+
+class ModelFromDramQueue(torch.nn.Module):
+
+    model_name = "model_op_src_from_dram_queue"
+
+    def __init__(self, operator, opname, shape, kwargs):
+        super(ModelFromDramQueue, self).__init__()
+        self.testname = "Element_wise_pytorch_binary_operator_" + opname + "_test_op_src_from_dram_queue"
+        self.operator = operator
+        self.opname = opname
+        self.shape = shape
+        self.kwargs = kwargs
+
+    def forward(self, x: torch.Tensor, y: torch.Tensor):
+        output = self.operator(x, y, **self.kwargs)
+        return output
+
+
+class ModelConstEvalPass(torch.nn.Module):
+
+    model_name = "model_op_src_const_eval_pass"
+
+    def __init__(self, operator, opname, shape, kwargs):
+        super(ModelConstEvalPass, self).__init__()
+        self.testname = "Element_wise_pytorch_binary_operator_" + opname + "_test_op_src_const_eval_pass"
+        self.operator = operator
+        self.opname = opname
+        self.shape = shape
+        self.kwargs = kwargs
+        
+        self.constant_shape = ShapeUtils.reduce_microbatch_size(shape)
+
+        # self.c1 = torch.rand(*self.constant_shape)
+        # self.c2 = torch.rand(*self.constant_shape)
+        self.c1 = (torch.rand(*self.constant_shape, requires_grad=False) - 0.5).detach()
+        self.c2 = (torch.rand(*self.constant_shape, requires_grad=False) - 0.5).detach()
+
+    def forward(self, x, y):
+        v1 = self.operator(self.c1, self.c2, **self.kwargs)
+        # v2 and v3 consume inputs
+        v2 = torch.add(x, y)
+        v3 = torch.add(v1, v2)
+        return v3
+
+
+
+def verify(
+    test_device: TestDevice,
+    model_type: Type[torch.nn.Module],
+    input_operator: str,
+    input_shape: TensorShape,
+    number_of_operands: int,
+    kwargs: Dict = {},
+    input_params: List[Dict] = [],
+    input_source_flag: InputSourceFlags = None,
+    dev_data_format: forge.DataFormat = None,
+    math_fidelity: forge.MathFidelity = None,
+):
+    '''Common verification function for all tests'''
+
+    xfail_test(input_operator, input_shape, model_type)
+
+    operator = getattr(torch, input_operator)
+
+    pytorch_model = model_type(operator=operator, opname=input_operator, shape=input_shape, kwargs=kwargs)
+    # forge_model = forge.PyTorchModule(pytorch_model.model_name, pytorch_model)
+
+    input_shapes = tuple([input_shape for _ in range(number_of_operands)])
+    logger.trace(f"***input_shapes: {input_shapes}")
+
+    VerifyUtils.verify(
+        model=pytorch_model,
+        test_device=test_device,
+        input_shapes=input_shapes,
+        input_params=input_params,
+        input_source_flag=input_source_flag,
+        dev_data_format=dev_data_format,
+        math_fidelity=math_fidelity,
+    )
+
+
+MODEL_TYPES = [
+    ModelFromAnotherOp,
+    ModelFromHost,
+    ModelFromDramQueue,
+    pytest.param(ModelConstEvalPass, marks = pytest.mark.skip(reason=FailingReasons.SEG_FAULT))
```

Comment:
It looks like const eval pass doesn't produce seg faults anymore after rebase on latest main. @nvukobratTT were there some improvements in last few days related const eval pass particularly?

Reply:
Yup, there were a few changes that targeted consteval :))